src/pages/products/Products.jsx

```diff
@@ -11,14 +12,12 @@ export function Products() {
     <>
       <Navbar />
       <main className="pos-rel">
-        <div className="filter-pos-size mob-hide">
+        <div className={`${styles.desktop__grid}`}>
```

Comment:
This is same as ```{styles.desktop__grid}```.

Reply:
I had kept it intentionally if lateron I would need to add regular CSS classes, but I understand it will cause confusion as well, hence will clean it up.